core/camera/src/main/java/com/google/jetpackcamera/core/camera/CameraXCameraUseCase.kt

```diff
@@ -197,6 +215,87 @@ constructor(
         imageCaptureExtender.setSessionCaptureCallback(captureCallback)
     }
 
+    @RequiresApi(Build.VERSION_CODES.P)
+    private fun getAllCamerasPropertiesJSONArray(): JSONArray {
+        val cameraManager =
+            this.application.baseContext.getSystemService(Context.CAMERA_SERVICE) as CameraManager
+        val result = JSONArray()
+        for (logicalCameraId in cameraManager.cameraIdList) {
+            val logicalCameraData = JSONObject()
+            logicalCameraData.put(
+                "logical-$logicalCameraId",
+                getCameraPropertiesJSONObject(logicalCameraId, cameraManager)
+            )
+            for (physicalCameraId in
+            cameraManager.getCameraCharacteristics(logicalCameraId).physicalCameraIds) {
+                logicalCameraData.put(
+                    "physical-$physicalCameraId",
+                    getCameraPropertiesJSONObject(physicalCameraId, cameraManager)
+                )
+            }
+            result.put(logicalCameraData)
+        }
+        return result
+    }
+
+    private fun getCameraPropertiesJSONObject(
+        cameraId: String,
```

Comment:
Can we move both of these methods to a separate file so we're not adding too much to this file? Maybe as free functions in a file called `DebugCameraInfo.kt`?